src/sentry/conf/server.py

```diff
@@ -1246,6 +1246,10 @@ def create_partitioned_queues(name):
 # Maximum content length for source files before we abort fetching
 SENTRY_SOURCE_FETCH_MAX_SIZE = 40 * 1024 * 1024
 
+# Maximum content length for cache value.
+# e.g. memcached defaults to 1MB
+SENTRY_CACHE_MAX_VALUE_SIZE = None
```

Comment:
Do we plan to use this setting anywhere else?
The description sounds a bit too generic now, maybe clarify the memcached example more?

Reply:
It can be used in different places as well but I mentioned that atm is only used for this purpose.